tests/everflow/test_everflow_testbed.py

```diff
@@ -50,7 +50,8 @@ def _partial_ptf_runner(setup_info, session_info, acl_stage, mirror_type,  expec
                    platform_dir="ptftests",
                    testname="everflow_tb_test.EverflowTest" if not test_name else test_name,
                    params=params,
-                   socket_recv_size=16384,
+                   socket_recv_size=10 * 1024 * 1024,
```

Comment:
@stepanblyschak can we have a named definition for these items?

Reply:
Fixed